bitnami/mysql/templates/master-statefulset.yaml

```diff
@@ -212,7 +212,7 @@ spec:
           {{- if or .Values.root.injectSecretsAsVolume }}
           volumeMounts:
             - name: mysql-credentials
-              mountPath: /opt/bitnami/mysql/secrets/
+              mountPath: /opt/bitnami/mysqld-exporter/secrets/
```

Comment:
Modify master statefulset's  volumeMounts mountPath value  is the same as ENV  parameter MYSQL_ROOT_PASSWORD_FILE